src/transpiling/mod.rs

```diff
@@ -1352,18 +1423,19 @@ for (let i = 0; i < testVariable >> 1; i++) callCount++;
 
   #[test]
   fn diagnostic_octal_and_leading_zero_num_literals() {
-    assert_eq!(get_diagnostic("077"), concat!(
+    // todo(dsherret): remove export {} after https://github.com/swc-project/swc/pull/9682
+    assert_eq!(get_diagnostic("077;\n\nexport {};"), concat!(
```

Comment:
I think it's ok to proceed without this for a little bit: https://github.com/denoland/deno_ast/issues/285